proof/crefine/RISCV64/Finalise_C.thy

```diff
@@ -637,6 +637,14 @@ lemma merge_overlapping_head_refill_ccorres:
   done
 
 (* FIXME RT: move to Corres_UL_C? *)
+lemma ccorres_to_vcg_Normal':
+  "\<lbrakk>ccorres_underlying srel \<Gamma> rrel xf arrel axf P P' [] a c\<rbrakk>
+   \<Longrightarrow> \<Gamma> \<turnstile> {s'. P s \<and> \<not> snd (a s) \<and> s' \<in> P' \<and> (s, s') \<in> srel} c UNIV"
```

Comment:
If this is a `'` variant, can the other one be derived from it (or vice versa)?

Reply:
I think I did try, and it didn't work, but I will give it another shot

Reply:
I was able to derive one from the other. Updated. Thanks